server/app/models/volume.rb

```diff
@@ -34,4 +34,16 @@ def name_for_service(service, instance_number)
     end
   end
 
+  def services
+    services = []
+    self.grid.grid_services.each do |service|
+      service.service_volumes.each do |sv|
```

Comment:
Well, not embedded docs themselves, but there's a way to query all services based on some attributes on the embedded doc values. Which actually work here nicely:
```
  def services
    self.grid.grid_services.where("service_volumes.volume_id" => self.id)
  end
```